presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -902,6 +902,19 @@ public void testExtractDistinctAggregationOptimizer()
         assertQuery("SELECT COUNT(tan(shippriority)), sum(DISTINCT orderkey) FROM orders");
 
         assertQuery("SELECT count(DISTINCT a), max(b) FROM (VALUES (row(1, 2), 3)) t(a, b)", "VALUES (1, 3)");
+
```

Comment:
Copied from https://github.com/prestodb/presto/pull/7417. The code fix in the legacy code has also been migrated to the new rule.